modules/manage/partials/monitor-redpanda.adoc

```diff
@@ -1,8 +1,6 @@
-Redpanda exports metrics through Prometheus endpoints for you to monitor system health and optimize system performance.
+Redpanda exports metrics through two Prometheus endpoints on the Admin API port (default: 9644) for you to monitor system health and optimize system performance.
```

Comment:
@wzzzrd86 @masapanda is it correct that we call out "Prometheus" here?

Reply:
Yes I think that is correct

Reply:
I think you solved this already, but just for the future, I don't think calling it "prometheus endpoints" is accurate. They're HTTP endpoints, through the Admin API, that responds with a prometheus format.